### ISSUE TITLE
Error Message: Cannot read properties of undefined 

### ISSUE THREAD

**Issue by RolandM99** [microsoft/vscode-azureappservice#2423]
Does this occur consistently? 
Repro steps:

1.
2.

Action: appService.Deploy
Error type: TypeError
Error Message: Cannot read properties of undefined (reading 'createClient')

Version: 0.24.7
OS: linux
OS Release: 5.19.0-35-generic
Product: Visual Studio Code
Product Version: 1.75.0
Language: en

<details>
<summary>Call Stack</summary>

```
extension.bundle.js:2:3774915
fulfilled extension.bundle.js:2:3773586
```

</details>

**Comment by alexweininger** [comment]
The fix for this bug has been merged and is waiting to be released in the Azure Resources extension. https://github.com/microsoft/vscode-azureresourcegroups/pull/642

**Comment by alexweininger** [comment]
This issue has been fixed by the [0.7.0 release of the Azure Resources extension](https://github.com/microsoft/vscode-azureresourcegroups/releases/tag/v0.7.0). Please update the Azure Resources extension to version 0.7.0.
